src/Helper/PaymentHelper.php

```diff
@@ -26,18 +26,46 @@ public function __construct(PaymentMethodRepositoryContract $paymentMethodReposi
 		$this->paymentMethodRepository = $paymentMethodRepository;
 	}
 
-	/**
-	 * @return int|null
-	 */
-	public function getPaymentMethodId()
+    /**
+     * @param string $paymentMethod
+     * @return int|null
+     */
+	public function getPaymentMethodId(string $paymentMethod = 'cc')
```

Comment:
would be cleaner to just add the param, where this function is called instead of making it a default in this case.
But works anyways.